cli/command/service/list.go

```diff
@@ -45,7 +45,9 @@ func runList(dockerCli *command.DockerCli, opts listOptions) error {
 	ctx := context.Background()
 	client := dockerCli.Client()
 
-	services, err := client.ServiceList(ctx, types.ServiceListOptions{Filters: opts.filter.Value()})
+	serviceFilters := opts.filter.Value()
+	serviceFilters.Add("runtime", string(swarm.RuntimeContainer))
```

Comment:
IIUC, this adds a new filter option to `docker service ls`, and the API for that command? If so, please add the new filter to the CLI docs; https://github.com/docker/docker/blob/master/docs/reference/commandline/service_ls.md#filtering, and the API history https://github.com/docker/docker/blob/master/docs/api/version-history.md#v129-api-changes